endpoints/openrtb2/sample-requests/disabled/good/partial.json

```diff
@@ -43,7 +43,7 @@
           "openx": {
             "unit": "539439964",
             "delDomain": "se-demo-d.openx.net",
-            "customFloor": 0.1,
+            "customFloor": "0.5",
             "customParams": {"foo": "bar"}
           }              
         }
```

Comment:
@laurb9 why was this file changed? Adapter change should be limited to Adapter package

Reply:
This file contains an OpenX adapter request, and we thought it would be best if it were updated to the most recent request format.

Reply:
could move this to openx specific exemplary test folder

Reply:
I can undo this particular line change if it blocks the review, but moving it to the openx test folder does not add any new use cases there. Removing samples from `openrtb2/sample-requests` is outside the scope of this PR, aside from that `openrtb2/sample-requests` contains requests for other adapters as well, so the file looks consistent with the intended purpose of that directory, not as an adapter-specific request, but an usage example for certain pbs configurations.